verification/curator-service/ui/src/components/SourceTable.tsx

```diff
@@ -331,18 +356,16 @@ class SourceTable extends React.Component<Props, SourceTableState> {
                                         onChange={(event): void =>
                                             props.onChange(event.target.value)
                                         }
-                                        defaultValue={props.value}
+                                        defaultValue={props.value || ''}
```

Comment:
What's the background for this change? Suppose this question also applies to having the empty string for `dateFilter.op`. Though in that case, we're adding an enum to the API, which is what's important. The empty string doesn't really cause any harm if we can prevent sending that data (replacing it with undefined instead) from the UI.

Reply:
The purpose is getting rid of warnings showing up in the console, undefined is not a valid value (must be a valid string). Did the same for the JSON/CSV select as well now we have no more warnings about those.